cli/src/Morphir/Web/DevelopApp.elm

```diff
@@ -1535,8 +1535,8 @@ viewHome model packageName packageDef =
         ]
 
 
-definitionList : Package.Definition () (Type ()) -> Model -> List FQName -> Element Msg
-definitionList packageDef model entrypoints =
+definitionList : PackageName -> Package.Definition () (Type ()) -> Model -> List FQName -> Element Msg
+definitionList packageName packageDef model entrypoints =
```

Comment:
does `packageName` get used?

Reply:
yes, on line 1578. It was originally using a different (wrong) packageName value, which was imported from another module rather then passed to the function. So that line did not need to change.

Reply:
Ah! got it. Thanks